### PR TITLE
Add specs synch notes 2020-11-05

### PR DESCRIPTION
**Pull request by warpfork**

### CODE REVIEW

File: meeting-notes/2020/2020-11-05--ipld-specs.md [hunk 1]
```diff
@@ -0,0 +1,199 @@
...
+
+Punted until next week; EAGAIN.
+
+
+Changing to CC-BY
+-----------------
+
+https://github.com/ipld/specs/issues/298
+
+We are a little concerned about licensing on specs... specs and docs are maybe a little different and we should be mindful on that.
```
Comment (rvagg) [inline meeting-notes/2020/2020-11-05--ipld-specs.md:187]: Mikeal had a comment here that was pretty important to record: Apache-2.0 includes some patent protective language that we miss out on by switching to a pure CC-BY.

FWIW this is a reason I've switched my default open source license in my own projects to Apache-2.0, this has become a live issue in the wild and MIT doesn't really cut it. Can recommend others follow suit.

File: meeting-notes/2020/2020-11-05--ipld-specs.md [hunk 2]
```diff
@@ -0,0 +1,199 @@
+specs synch 2020.11.05
```
Comment (rvagg) [inline meeting-notes/2020/2020-11-05--ipld-specs.md:1]: I'd like to see attendees list in these documents, it's important for understanding the nature of "consensus". There are some cases where discussion can become meaningless where a single individual is absent and has the ability to nullify something wither objection.

Also, we really should make these meetings open for attendance, they're much more meaty than our syncs.

**Comment by rvagg** [comment]
Going to record further thoughts that I noted in a more ephemeral media on the subject of constraining the `"/"` map key for DAG-JSON. I've suggested that we could introduce an escaping mechanism of sorts to allow for the encoding of data that uses `"/"` for things other than bytes and CID because we've all already acknowledged that this character is not uncommon. One possibility is copying the [UTF8-C8 approach](https://docs.raku.org/language/unicode) and using the `0x10FFFD` private codepoint as an escape (of sorts).

> the point being that it would nice to have the domain of "possible" as broad as we can so that there isn't this space where we throw up our hands and say "nope". For test fixtures I'm a little less concerned, but for human-readable printing of IPLD data from other codecs, what does dag-json do when it encounters a map key "/"?  it's an error, which is just crappy if you're writing tooling that's doing inspection or other kind of printing. ipfs dag get <thing> should be able to show thing regardless of codec. This rule means it would error if ipfs dag get implements proper dag-json and it encounters "/"
> 
> and the point of a get-out-of-jail here is to stretch the domain slightly to make it possible. 0x10fffd + '/' as a special escape would make it possible. "􏿽/", it's not pretty, but it's possible and it would round-trip.
> 
> or maybe it's "􏿽2f" if we follow strict utf8-c8 rules
> 
> one special-case rule for dag-json to make this work, that byte string could even be hard-wired as a constant in our encoders/decoders to make this work
> 
> it does rule out "􏿽2f" as a valid map key, but expected number of real-world maps with that as a key is vanishingly small, compared to "/" which is quite common, so we're squeezing the domain down here
> 
> I think I like "􏿽/", no need to pretend to be utf8-c8 here and that's still approximately human readable

I'll probably turn this into a specs PR so it can be discussed further.

**Review by vmx — APPROVED** [review]

File: meeting-notes/2020/2020-11-05--ipld-specs.md (hunk 2 above)
Comment (vmx) [inline meeting-notes/2020/2020-11-05--ipld-specs.md:1]: Nit (not really important, just for future reference). ISO dates are written with a dash in between. In Germany this dot notation is usually used for dates, but there it's written as 05.11.2020.

**Pull request merged**